services/userServices.js

```diff
@@ -6,7 +6,7 @@ const addUserService = async (user) => {
   return resData;
```

Comment:
Are we getting newly created data inside data key?
if so instead of using both await and then, can we change it like 
const resData = await userTemplateCopy.create(user);
return resData.data;

Reply:
Yes, Ram, I made it look complex. I'll Change.

Reply:
Valid point! Changed Ram!